frontend/src/components/events/sponsors/index.jsx

```diff
@@ -44,8 +44,8 @@ class Sponsors extends Component {
       <div className="sponsor-gallery">
         {sponsorsHeading}
         <Container>
-          {urls.map(url => (
-            <img src={url} className="logo-picture" alt="sponsor logo" />
+          {urls.map((url, i) => (
```

Comment:
Typically using index for react keys is an anti pattern. In this case it's fine because the urls are not likely to be reordered or filtered on the fly. For the future I would recommend against this but now 👍 

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
yaaa thats a good call but in this case we dont have an id so 🤷‍♂ we could potentially use the url